scaaml/capture/aes/scope.py

```diff
@@ -14,6 +14,21 @@ def __init__(self, gain: int, samples: int, offset: int, clock: int,
           offset: Number of samples to wait before starting recording data.
           clock: CLKGEN output frequency (in Hz).
           sample_rate: Clock source for cw.ClockSettings.adc_src.
+          _: SScope is expected to by initialized with capture_info which may
+            contain more keys. Thus we can ignore the rest of keyword
```

Comment:
I don't find `more keys` easy to understand in this context. Either we need to add that `capture_info` is a `dict` or use `keywords`, like you did on the next sentence. WDYT?

Reply:
Much better. Thanks